src/components/reusable/header/FlowHeader.tsx

```diff
@@ -33,10 +34,16 @@ export default function FlowHeader({
     <>
       <Header>
         <Container>
-          <Icon type="button" onClick={callback}>
-            {iconType === IconType.back ? <ArrowBackFilled /> : <CloseIcon />}
-          </Icon>
-          <Text>{text}</Text>
+          <>
+            {iconType !== IconType.empty ? (
+              <Icon type="button" onClick={callback}>
+                {iconType === IconType.back ? <ArrowBackFilled /> : <CloseIcon />}
+              </Icon>
+            ) : (
+              <EmptyIcon />
```

Comment:
I think here empty icon is redundant, I suggest making iconType in props optional and if it was passed then show icon, else not, because this is hard to read somehow

Reply:
This way we also won't need empty icon component

Reply:
I belive it's not a better solution because if we don't pass icon layout will break, so we will need to check whether it's passed and make styles for header dynamically which is not really easier than passing prewritten empty element.

Reply:
Fixed)